google/services/apikeys/resource_apikeys_key.go

```diff
@@ -132,6 +132,13 @@ func ApikeysKeyRestrictionsSchema() *schema.Resource {
 				Elem:        ApikeysKeyRestrictionsIosKeyRestrictionsSchema(),
 			},
 
+			"keepers": {
```

Comment:
This service doesn't look like it's being generated in magic-modules, as far as I can tell: https://github.com/GoogleCloudPlatform/magic-modules/blob/f7457595c9214bdafd57a70898b65b948105ed0e/mmv1/third_party/terraform/provider/provider.go.erb#L468-L474